### PR TITLE
Allow setting CI build interval via client

### PR DESCRIPTION
**Pull request by nedSaf**
#187

### CODE REVIEW

**Comment by nedSaf** [comment]
Update: 
localhost_9000___repos_22

**Comment by nedSaf** [comment]
Update:
interval

**Comment by nedSaf** [comment]
@HelenaEksler Could you review this for me please ?

File: client/app/scripts/controllers/repo.js [hunk 1]
```diff
@@ -21,6 +21,16 @@ angular.module('clientApp')
       'done': 'Done'
     };
 
+    // List of build statuses.
```
Comment (HelenaEksler) [inline client/app/scripts/controllers/repo.js]: Comment looks wrong. I think it should be something like "List of build possible intervals"

File: client/app/scripts/controllers/repo.js [hunk 2]
```diff
@@ -21,6 +21,16 @@ angular.module('clientApp')
...
+    $scope.intervals = {
+      180: '3 Min',
+      3600: '1 Hour',
+      86400: '1 Day'
+    };
+
+    // Show the success icon when there's a response from the backend.
+    $scope.responseClass = false;
```
Comment (HelenaEksler) [inline client/app/scripts/controllers/repo.js]: I think the variable name should be `responseStatus`, how do you think?

**Comment by nedSaf** [comment]
@HelenaEksler Fixed, Thanks.

@amitaibu Ready for code review.

File: client/app/scripts/controllers/repo.js [hunk 3]
```diff
@@ -33,6 +43,30 @@ angular.module('clientApp')
...
+      var params = {
+        'interval': $scope.build.interval
+      };
+
+      Builds
+        .update($scope.build.id, 'ci_build', params)
+        .then(function() {
+          $scope.responseStatus = true;
+
+          // Hide the success icon after 5 seconds of receiving the response.
```
Comment (amitaibu) [inline client/app/scripts/controllers/repo.js]: 3 seconds?

Comment (nedSaf) [inline client/app/scripts/controllers/repo.js]: Sorry, yes, Thanks for fixing.

**Comment by amitaibu** [comment]
Merged, thanks.

**Pull request closed without merging**